File: CAPITULO 01/C++/ex7.cpp
#include <stdio.h>
int main()
{
	//continuo tendo problemas em exibir o resultado
	double sal,grat,imp,sal_receber;
	printf("Digite o salario base do funcionario :");
	scanf("%d%*c",&sal);
	grat = 50;
	sal_receber = sal+grat +(sal*0.10);
	printf("O salario a receber pelo funcionario e de : %2.5f");
	getchar();
	return 0;
	
}

File: CAPITULO 01/Java/ex7.java
import java.io.*;
import java.util.*;
import java.text.*;
public class ex7{
	public static void main(String[]args){
		double sal_base, grat, desc, sal_receber;
		Scanner dado;
		dado =  new Scanner (System.in);
		System.out.println("Digite o valor do salario base do funcionario :");
		sal_base = dado.nextDouble();
		grat = 50;
		desc = sal_base * 0.10;
		sal_receber = sal_base + grat + desc;
		DecimalFormat cs;
		cs = new DecimalFormat("0.00");
		System.out.println("O valor do salario do funcionario e de "+cs.format(sal_receber));
	}
}